packages/web-components/fast-foundation/docs/api-report.md

```diff
@@ -881,7 +881,15 @@ export interface ElementDefinitionContext {
 }
 
 // @public
-export type ElementDisambiguationCallback = (nameAttempt: string, typeAttempt: Constructable, existingType: Constructable) => string | null;
+export const ElementDisambiguation: Readonly<{
+    definitionCallbackOnly: null;
+    ignoreDuplicate: symbol;
+}>;
+
+// Warning: (ae-forgotten-export) The symbol "ElementDisambiguationResult" needs to be exported by the entry point index.d.ts
```

Comment:
Does ElementDisambiguationResult need to be private?

Reply:
I don't think it needs to be. Let me export that and add some docs.